cloud/deploys/bin/azure/key-vault.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+#######################################
+# Create resource group
+# Arguments:
+#   1: The resource group name for the key vault
+#   2: The location of the key vault
```

Comment:
question (blocking): is the location the same as the region e.g. `EastUS`?

Reply:
Yup. Clarified this in the comments.